torch_sim/optimizers.py

```diff
@@ -33,6 +33,28 @@
 MdFlavor = Literal["vv_fire", "ase_fire"]
 vv_fire_key, ase_fire_key = get_args(MdFlavor)
 
+md_atom_attributes = (*SimState._atom_attributes, "forces", "velocities")  # noqa: SLF001
```

Comment:
Can we just redefine this in integrators and optimizers instead of importing it in integrators? Feels cleaner as a private method. I don't like creating a dependency between integrators and optimizers